src/components/pages/dashboard/links.ts

```diff
@@ -66,9 +66,9 @@ const links = [
       },
       {
         title: "💼 ROBUST ODK",
-        description: "View aggregated data on ODK",
+        description: "Login to ROBUST ODK",
```

Comment:
can we put this in translation file?

Reply:
issues using translation requires more work